src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -74,6 +100,35 @@ public override DbCommand GetCommand(DbConnection connection)
             return command;
         }
 
+        /// <summary>
+        /// Runs a query using the where clause to determine if duplicates are found (causes issues when deleting).
+        /// </summary>
+        private bool checkWhereDuplicate(WhereClause where, string input, DbConnection connection)
```

Comment:
please following .Net naming conventions, which says functions should be PascalCase, so `CheckWhereDuplicate` in this case.  Though personally I think a name like `CheckForDuplicateDeleteRows` would be more "self documenting".